neptune/internal/backends/offline_backend.py

```diff
@@ -131,14 +131,5 @@ def download_data(self, project, path, destination):
 
 class NoopObject(object):
 
-    def __getattribute__(self, item):
-        return NoopObject()
-
-    def __call__(self, *args, **kwargs):
-        return NoopObject()
-
-    def __enter__(self):
-        return NoopObject()
-
-    def __exit__(self, exc_type, exc_val, exc_tb):
-        pass
+    def __getattr__(self, item):
+        return None
```

Comment:
I didn't see the point of `NoopObject` being returned from `__getattr__` so I made it return `None`. I might be mistaken or be missing something though.